docs/topics/mta-rn-upgrade-cli.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-cli_{context}"]
+= Setting the parameter in the {ocp-short} web console after upgrading to version {ProductVersion}
+
+You can set the value of `rwx_supported` in the {ocp-short} web console after you upgrade to {ProductShortName} {ProductVersion}
```

Comment:
You can set the value of `rwx_supported` in the {ocp-short}  console after you upgrade to {ProductShortName} {ProductVersion}